calico-cloud_versioned_docs/version-19-1/get-started/connect/install-helm.mdx

```diff
@@ -0,0 +1,41 @@
+---
+description: Steps to connect your cluster to Calico Cloud.
+title: Install using Helm
+---
+
+# Connect a cluster to Calico Cloud using Helm
```

Comment:
@jaderhs Still a draft, but this is where I'm at. 

As we discussed, I'm adding your feature as an optional step in the regular install procedure. To be honest, it still feels clogged. But the alternative -- writing a separate procedure just for those flags -- seems more out of place. I suspect the balance will tip the other direction when this feature has more options.